### ISSUE TITLE
[Bug] Some weird Giant Willow growth problem (floating torches under the tree and a bunch of torch item dropped at tree spawn).

### ISSUE THREAD

**Issue by SatDog92** [quiqueck/BetterNether#65]
### What happened?

Apparently, i just found out that there is a small chance that a giant willow can grow in place of a small one. The problem is that the tree growth is very weird: as soon as the giant willow pops up, a strong sound of broken blocks is played, and the tree looks like something seen in [this picture](https://user-images.githubusercontent.com/56725716/195183507-42ade8f6-4c71-4e77-b2e8-235dc9a039c7.png).

The broken blocks are most likely the willow torches that broke on tree spawn for some reason, and the torch part of the willow branches are floating in air.

This is a minor bug, but still pretty jarring.

### BetterNether

7.1.0

### BCLib

2.1.0

### Fabric API

0.60.0

### Fabric Loader

0.14.9

### Minecraft

1.19.2

### Relevant log output

```shell
[21:26:44] [main/INFO]: Loading Minecraft 1.19.2 with Fabric Loader 0.14.9
[21:26:45] [main/INFO]: Loading 53 mods:
	- bclib 2.1.0
	- betternether 7.1.0
	- fabric-api 0.60.0+1.19.2
	- fabric-api-base 0.4.11+e62f51a390 via fabric-api
	- fabric-api-lookup-api-v1 1.6.9+9ff28f4090 via fabric-api
	- fabric-biome-api-v1 9.0.17+aeb40ebe90 via fabric-api
	- fabric-blockrenderlayer-v1 1.1.20+aeb40ebe90 via fabric-api
	- fabric-client-tags-api-v1 1.0.1+b35fea8390 via fabric-api
	- fabric-command-api-v1 1.2.11+f71b366f90 via fabric-api
	- fabric-command-api-v2 2.1.7+0c17ea9690 via fabric-api
	- fabric-commands-v0 0.2.28+df3654b390 via fabric-api
	- fabric-containers-v0 0.1.34+df3654b390 via fabric-api
	- fabric-content-registries-v0 3.2.3+aeb40ebe90 via fabric-api
	- fabric-convention-tags-v1 1.1.1+7cd20a1490 via fabric-api
	- fabric-crash-report-info-v1 0.2.5+aeb40ebe90 via fabric-api
	- fabric-data-generation-api-v1 5.1.8+a680b9b490 via fabric-api
	- fabric-dimensions-v1 2.1.30+aeb40ebe90 via fabric-api
	- fabric-entity-events-v1 1.4.18+9ff28f4090 via fabric-api
	- fabric-events-interaction-v0 0.4.28+aeb40ebe90 via fabric-api
	- fabric-events-lifecycle-v0 0.2.28+df3654b390 via fabric-api
	- fabric-game-rule-api-v1 1.0.21+aeb40ebe90 via fabric-api
	- fabric-item-api-v1 1.5.7+35a03c4390 via fabric-api
	- fabric-item-groups-v0 0.3.28+aeb40ebe90 via fabric-api
	- fabric-key-binding-api-v1 1.0.20+aeb40ebe90 via fabric-api
	- fabric-keybindings-v0 0.2.18+df3654b390 via fabric-api
	- fabric-lifecycle-events-v1 2.1.2+aeb40ebe90 via fabric-api
	- fabric-loot-api-v2 1.1.3+83a8659290 via fabric-api
	- fabric-loot-tables-v1 1.1.6+9e7660c690 via fabric-api
	- fabric-message-api-v1 5.0.3+176380a290 via fabric-api
	- fabric-mining-level-api-v1 2.1.14+33fbc73890 via fabric-api
	- fabric-models-v0 0.3.17+aeb40ebe90 via fabric-api
	- fabric-networking-api-v1 1.2.4+5eb68ef290 via fabric-api
	- fabric-networking-v0 0.3.21+df3654b390 via fabric-api
	- fabric-object-builder-api-v1 4.0.11+aeb40ebe90 via fabric-api
	- fabric-particles-v1 1.0.10+aeb40ebe90 via fabric-api
	- fabric-registry-sync-v0 0.9.24+aeb40ebe90 via fabric-api
	- fabric-renderer-api-v1 1.0.10+aeb40ebe90 via fabric-api
	- fabric-renderer-indigo 0.6.12+aeb40ebe90 via fabric-api
	- fabric-renderer-registries-v1 3.2.20+df3654b390 via fabric-api
	- fabric-rendering-data-attachment-v1 0.3.14+aeb40ebe90 via fabric-api
	- fabric-rendering-fluids-v1 3.0.7+aeb40ebe90 via fabric-api
	- fabric-rendering-v0 1.1.22+df3654b390 via fabric-api
	- fabric-rendering-v1 1.10.16+aeb40ebe90 via fabric-api
	- fabric-resource-conditions-api-v1 2.0.11+e62f51a390 via fabric-api
	- fabric-resource-loader-v0 0.6.1+aeb40ebe90 via fabric-api
	- fabric-screen-api-v1 1.0.26+f05effaf90 via fabric-api
	- fabric-screen-handler-api-v1 1.2.13+9ff28f4090 via fabric-api
	- fabric-textures-v0 1.0.20+aeb40ebe90 via fabric-api
	- fabric-transfer-api-v1 2.0.11+83a8659290 via fabric-api
	- fabric-transitive-access-wideners-v1 1.3.0+42d99c3290 via fabric-api
	- fabricloader 0.14.9
	- java 17
	- minecraft 1.19.2
[21:26:47] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/Utente/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[21:26:47] [main/INFO]: Compatibility level set to JAVA_17
[21:26:51] [main/WARN]: Configuration conflict: there is more than one oshi.properties file on the classpath
[21:26:52] [main/WARN]: Configuration conflict: there is more than one oshi.architecture.properties file on the classpath
[21:26:55] [main/INFO]: [bclib] Added Config bclib.generator to auto sync (file hash)
[21:26:55] [main/INFO]: [bclib] Added Config bclib.main to auto sync (content diff)
[21:26:55] [main/INFO]: [bclib] Added Config bclib.recipes to auto sync (file hash)
[21:26:55] [main/INFO]: [bclib] Added Config bclib.biomes to auto sync (file hash)
[21:27:01] [main/INFO]: Building optimized datafixer
[21:27:05] [main/INFO]: [bclib] Added Config betternether.main to auto sync (file hash)
[21:27:05] [main/INFO]: [bclib] Added Config betternether.generator to auto sync (file hash)
[21:27:05] [main/INFO]: [bclib] Added Config betternether.blocks to auto sync (file hash)
[21:27:05] [main/INFO]: [bclib] Added Config betternether.items to auto sync (file hash)
[21:27:05] [main/INFO]: [bclib] Added Config betternether.biomes to auto sync (file hash)
[21:27:05] [main/INFO]: [bclib] Added Config betternether.mobs to auto sync (file hash)
[21:27:05] [main/INFO]: [bclib] Added Config betternether.recipes to auto sync (file hash)
[21:27:07] [Render thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[21:27:08] [Render thread/INFO]: Setting user: SaturnoSomaro92
[21:27:08] [Render thread/INFO]: [betternether] =^..^=    BetterNether for 1.19    =^..^=
[21:27:09] [Thread-1/INFO]: [bclib] Received Version Info for minecraft=1.19.2, loader=fabric
[21:27:09] [Thread-1/INFO]: [bclib]  - bclib:2.1.0
[21:27:09] [Thread-1/INFO]: [bclib]  - betternether:7.1.0
[21:27:10] [Render thread/INFO]: [Indigo] Registering Indigo renderer!
[21:27:10] [Render thread/INFO]: Backend library: LWJGL version 3.3.1 SNAPSHOT
[21:27:13] [Render thread/INFO]: Narrator library for x64 successfully loaded
[21:27:13] [Render thread/INFO]: Reloading ResourceManager: Default, MySplash.zip, Fabric Mods (BCLib, Better Nether, Fabric API, Fabric API Base, Fabric API Lookup API (v1), Fabric Biome API (v1), Fabric BlockRenderLayer Registration (v1), Fabric Client Tags, Fabric Command API (v1), Fabric Command API (v2), Fabric Commands (v0), Fabric Containers (v0), Fabric Content Registries (v0), Fabric Convention Tags, Fabric Crash Report Info (v1), Fabric Data Generation API (v1), Fabric Dimensions API (v1), Fabric Entity Events (v1), Fabric Events Interaction (v0), Fabric Events Lifecycle (v0), Fabric Game Rule API (v1), Fabric Item API (v1), Fabric Item Groups (v0), Fabric Key Binding API (v1), Fabric Key Bindings (v0), Fabric Lifecycle Events (v1), Fabric Loot API (v2), Fabric Loot Tables (v1), Fabric Message API (v1), Fabric Mining Level API (v1), Fabric Models (v0), Fabric Networking API (v1), Fabric Networking (v0), Fabric Object Builder API (v1), Fabric Particles (v1), Fabric Registry Sync (v0), Fabric Renderer API (v1), Fabric Renderer - Indigo, Fabric Renderer Registries (v1), Fabric Rendering Data Attachment (v1), Fabric Rendering Fluids (v1), Fabric Rendering (v0), Fabric Rendering (v1), Fabric Resource Conditions API (v1), Fabric Resource Loader (v0), Fabric Screen API (v1), Fabric Screen Handler API (v1), Fabric Textures (v0), Fabric Transfer API (v1), Fabric Transitive Access Wideners (v1), Fabric Loader), Saturn the Donkey's Villagers.zip
[21:27:17] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[21:27:17] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[21:27:17] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on LEN D32q-20B (NVIDIA High Definition Audio)
[21:27:17] [Render thread/INFO]: Sound engine started
[21:27:17] [Render thread/INFO]: Created: 2048x1024x4 minecraft:textures/atlas/blocks.png-atlas
[21:27:17] [Render thread/INFO]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[21:27:17] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[21:27:17] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[21:27:17] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[21:27:17] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[21:27:17] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[21:27:19] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[21:27:19] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[21:27:19] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[21:27:19] [Render thread/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[21:27:24] [Render thread/INFO]: Deleting level kokokoko
[21:27:24] [Render thread/INFO]: Attempt 1...
[21:27:24] [Worker-Bootstrap-7/INFO]: 4283 Datafixer optimizations took 22705 milliseconds
[21:27:25] [Render thread/INFO]: Deleting level gegegege
[21:27:25] [Render thread/INFO]: Attempt 1...
[21:27:27] [Render thread/INFO]: Deleting level New World
[21:27:27] [Render thread/INFO]: Attempt 1...
[21:27:31] [Render thread/INFO]: Loaded 7 recipes
[21:27:32] [Render thread/INFO]: Loaded 1605 advancements
[21:27:39] [Render thread/INFO]: Loaded 7 recipes
[21:27:39] [Render thread/INFO]: Loaded 1605 advancements
[21:27:46] [Render thread/INFO]: [DataFixerAPI] Applying Patch{betternether:5.3.5:21020677}
[21:27:46] [Render thread/INFO]: [DataFixerAPI] Updating Patch-Level for 'betternether' from 0.0.0 to 5.3.5
[21:27:47] [Render thread/INFO]: Loaded 7 recipes
[21:27:47] [Render thread/INFO]: Loaded 1605 advancements
[21:27:48] [Render thread/INFO]: Applied 22 biome modifications to 22 of 85 new biomes in 25.36 ms
[21:27:48] [Server thread/INFO]: Starting integrated minecraft server version 1.19.2
[21:27:48] [Server thread/INFO]: Generating keypair
[21:27:48] [Server thread/INFO]: [bclib] 
Apply Modifications for minecraft:overworld
    class_4766 (7b30a6a2)
        biomes     = 52
        namespaces = minecraft
[21:27:48] [Server thread/INFO]: [bclib] 
Apply Modifications for minecraft:the_nether
    BCLib - Nether BiomeSource (447a2edf)
        biomes     = 27
        namespaces = betternether, minecraft
        seed       = 6872588633423707648
        height     = 128
        config     = BCLibNetherBiomeSourceConfig{mapVersion=hex}
[21:27:48] [Server thread/INFO]: [bclib] 
Apply Modifications for minecraft:the_end
    BCLib - The End BiomeSource (2ce4dcc)
        biomes     = 5
        namespaces = minecraft
        seed       = 6872588633423707648
        height     = 128
        deciders   = 0
        config     = BCLEndBiomeSourceConfig{mapVersion=hex, generatorVersion=paulevs, withVoidBiomes=true, innerVoidRadiusSquared=1048576, voidBiomesSize=256, centerBiomesSize=256, landBiomesSize=256, barrensBiomesSize=256}
[21:27:48] [Server thread/INFO]: [bclib] World Dimensions: 
 - minecraft:overworld: 
     net.minecraft.class_3754@48325299 
     class_4766 (7b30a6a2)
         biomes     = 52
         namespaces = minecraft
 - minecraft:the_nether: 
     BCLib - Chunk Generator (64c8be74) 
     BCLib - Nether BiomeSource (447a2edf)
         biomes     = 27
         namespaces = betternether, minecraft
         seed       = 6872588633423707648
         height     = 128
         config     = BCLibNetherBiomeSourceConfig{mapVersion=hex}
 - minecraft:the_end: 
     BCLib - Chunk Generator (7727e85) 
     BCLib - The End BiomeSource (2ce4dcc)
         biomes     = 5
         namespaces = minecraft
         seed       = 6872588633423707648
         height     = 128
         deciders   = 0
         config     = BCLEndBiomeSourceConfig{mapVersion=hex, generatorVersion=paulevs, withVoidBiomes=true, innerVoidRadiusSquared=1048576, voidBiomesSize=256, centerBiomesSize=256, landBiomesSize=256, barrensBiomesSize=256}
[21:28:03] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[21:28:03] [Render thread/INFO]: Preparing spawn area: 0%
[21:28:03] [Render thread/INFO]: Preparing spawn area: 0%
[21:28:04] [Render thread/INFO]: Preparing spawn area: 0%
[21:28:04] [Render thread/INFO]: Preparing spawn area: 1%
[21:28:05] [Render thread/INFO]: Preparing spawn area: 2%
[21:28:05] [Render thread/INFO]: Preparing spawn area: 3%
[21:28:06] [Render thread/INFO]: Preparing spawn area: 7%
[21:28:06] [Render thread/INFO]: Preparing spawn area: 10%
[21:28:07] [Render thread/INFO]: Preparing spawn area: 15%
[21:28:07] [Render thread/INFO]: Preparing spawn area: 19%
[21:28:08] [Render thread/INFO]: Preparing spawn area: 23%
[21:28:08] [Render thread/INFO]: Preparing spawn area: 27%
[21:28:09] [Render thread/INFO]: Preparing spawn area: 30%
[21:28:10] [Render thread/INFO]: Preparing spawn area: 34%
[21:28:10] [Render thread/INFO]: Preparing spawn area: 34%
[21:28:10] [Render thread/INFO]: Preparing spawn area: 36%
[21:28:11] [Render thread/INFO]: Preparing spawn area: 41%
[21:28:11] [Render thread/INFO]: Preparing spawn area: 45%
[21:28:12] [Render thread/INFO]: Preparing spawn area: 48%
[21:28:12] [Render thread/INFO]: Preparing spawn area: 53%
[21:28:13] [Render thread/INFO]: Preparing spawn area: 57%
[21:28:13] [Render thread/INFO]: Preparing spawn area: 62%
[21:28:14] [Render thread/INFO]: Preparing spawn area: 67%
[21:28:14] [Render thread/INFO]: Preparing spawn area: 72%
[21:28:15] [Render thread/INFO]: Preparing spawn area: 76%
[21:28:15] [Render thread/INFO]: Preparing spawn area: 81%
[21:28:16] [Render thread/INFO]: Preparing spawn area: 87%
[21:28:16] [Render thread/INFO]: Preparing spawn area: 92%
[21:28:17] [Render thread/INFO]: Preparing spawn area: 98%
[21:28:17] [Render thread/INFO]: Time elapsed: 14052 ms
[21:28:17] [Server thread/INFO]: Changing view distance to 16, from 10
[21:28:17] [Server thread/INFO]: Changing simulation distance to 12, from 0
[21:28:21] [Server thread/INFO]: SaturnoSomaro92[local:E:92303ada] logged in with entity id 209 at (359.5, 78.0, -246.5)
[21:28:21] [Server thread/INFO]: SaturnoSomaro92 joined the game
[21:28:22] [Render thread/INFO]: [bclib] Sending hello to server.
[21:28:22] [Server thread/INFO]: [bclib] Received Hello from Client. (server=2.1.0, client=2.1.0)
[21:28:22] [Server thread/INFO]: [bclib] Auto-Sync is disabled for Singleplayer worlds.
[21:28:22] [Render thread/INFO]: Loaded 0 advancements
[21:28:24] [Server thread/INFO]: Saving and pausing game...
[21:28:24] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:28:24] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:28:24] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:28:57] [Server thread/INFO]: Saving and pausing game...
[21:28:57] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:28:57] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:28:57] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:28:57] [Server thread/INFO]: SaturnoSomaro92 lost connection: Disconnected
[21:28:57] [Server thread/INFO]: SaturnoSomaro92 left the game
[21:28:57] [Server thread/INFO]: Stopping singleplayer server as player logged out
[21:28:57] [Server thread/INFO]: Stopping server
[21:28:57] [Server thread/INFO]: Saving players
[21:28:57] [Server thread/INFO]: Saving worlds
[21:28:59] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:29:02] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:29:02] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:29:02] [Server thread/INFO]: ThreadedAnvilChunkStorage (Juliet Willow): All chunks are saved
[21:29:02] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[21:29:02] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[21:29:02] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[21:29:09] [Render thread/INFO]: Deleting level Juliet Willow
[21:29:09] [Render thread/INFO]: Attempt 1...
[21:29:10] [Render thread/INFO]: Loaded 7 recipes
[21:29:10] [Render thread/INFO]: Loaded 1605 advancements
[21:29:12] [Render thread/INFO]: Loaded 7 recipes
[21:29:12] [Render thread/INFO]: Loaded 1605 advancements
[21:29:26] [Render thread/INFO]: [DataFixerAPI] Applying Patch{betternether:5.3.5:21020677}
[21:29:26] [Render thread/INFO]: [DataFixerAPI] Updating Patch-Level for 'betternether' from 0.0.0 to 5.3.5
[21:29:27] [Render thread/INFO]: Loaded 7 recipes
[21:29:27] [Render thread/INFO]: Loaded 1605 advancements
[21:29:28] [Render thread/INFO]: Applied 22 biome modifications to 22 of 85 new biomes in 671.2 ?s
[21:29:28] [Server thread/INFO]: Starting integrated minecraft server version 1.19.2
[21:29:28] [Server thread/INFO]: Generating keypair
[21:29:28] [Server thread/INFO]: [bclib] 
Apply Modifications for minecraft:overworld
    class_1992 (20f0bfb6)
        biomes     = 1
        namespaces = minecraft
[21:29:28] [Server thread/INFO]: [bclib] 
Apply Modifications for minecraft:the_nether
    class_4766 (9307350)
        biomes     = 15
        namespaces = minecraft, betternether
[21:29:28] [Server thread/INFO]: [bclib] 
Apply Modifications for minecraft:the_end
    class_2169 (71754d24)
        biomes     = 5
        namespaces = minecraft
[21:29:28] [Server thread/INFO]: [bclib] World Dimensions: 
 - minecraft:overworld: 
     net.minecraft.class_2897@65f225cc 
     class_1992 (20f0bfb6)
         biomes     = 1
         namespaces = minecraft
 - minecraft:the_nether: 
     net.minecraft.class_3754@545d4516 
     class_4766 (9307350)
         biomes     = 15
         namespaces = minecraft, betternether
 - minecraft:the_end: 
     net.minecraft.class_3754@2f3cf2b5 
     class_2169 (71754d24)
         biomes     = 5
         namespaces = minecraft
[21:29:28] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[21:29:28] [Render thread/INFO]: Preparing spawn area: 0%
[21:29:28] [Render thread/INFO]: Preparing spawn area: 3%
[21:29:29] [Render thread/INFO]: Preparing spawn area: 13%
[21:29:29] [Render thread/INFO]: Preparing spawn area: 25%
[21:29:30] [Render thread/INFO]: Preparing spawn area: 36%
[21:29:30] [Render thread/INFO]: Preparing spawn area: 49%
[21:29:31] [Render thread/INFO]: Preparing spawn area: 59%
[21:29:31] [Render thread/INFO]: Preparing spawn area: 73%
[21:29:32] [Render thread/INFO]: Preparing spawn area: 83%
[21:29:32] [Render thread/INFO]: Preparing spawn area: 95%
[21:29:33] [Server thread/INFO]: Changing view distance to 16, from 10
[21:29:33] [Server thread/INFO]: Changing simulation distance to 12, from 0
[21:29:33] [Render thread/INFO]: Time elapsed: 4678 ms
[21:29:33] [Server thread/INFO]: SaturnoSomaro92[local:E:6a79de47] logged in with entity id 1912 at (8.5, -60.0, -4.5)
[21:29:33] [Server thread/WARN]: [bclib] Server changed!
[21:29:33] [Server thread/INFO]: SaturnoSomaro92 joined the game
[21:29:33] [Render thread/INFO]: [bclib] Sending hello to server.
[21:29:33] [Server thread/INFO]: [bclib] Received Hello from Client. (server=2.1.0, client=2.1.0)
[21:29:33] [Server thread/INFO]: [bclib] Auto-Sync is disabled for Singleplayer worlds.
[21:29:33] [Render thread/INFO]: Loaded 0 advancements
[21:29:35] [Server thread/INFO]: Saving and pausing game...
[21:29:35] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:29:35] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:29:35] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:30:01] [Render thread/INFO]: Loaded 3 advancements
[21:30:18] [Render thread/INFO]: Loaded 5 advancements
[21:31:58] [Server thread/INFO]: [SaturnoSomaro92: Set own game mode to Spectator Mode]
[21:31:58] [Render thread/INFO]: [System] [CHAT] Set own game mode to Spectator Mode
[21:32:13] [Server thread/INFO]: [SaturnoSomaro92: Set own game mode to Creative Mode]
[21:32:13] [Render thread/INFO]: [System] [CHAT] Set own game mode to Creative Mode
[21:32:14] [Server thread/INFO]: [SaturnoSomaro92: Set own game mode to Spectator Mode]
[21:32:14] [Render thread/INFO]: [System] [CHAT] Set own game mode to Spectator Mode
[21:32:27] [Render thread/INFO]: [System] [CHAT] Saved screenshot as 2022-10-11_21.32.27.png
[21:32:36] [Server thread/INFO]: [SaturnoSomaro92: Set own game mode to Creative Mode]
[21:32:37] [Render thread/INFO]: [System] [CHAT] Set own game mode to Creative Mode
[21:32:43] [Server thread/INFO]: Saving and pausing game...
[21:32:43] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:32:43] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:32:43] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:36:19] [Server thread/INFO]: Saving and pausing game...
[21:36:19] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:36:19] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:36:19] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:41:02] [Server thread/INFO]: Saving and pausing game...
[21:41:02] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:41:02] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:41:02] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:42:42] [Server thread/INFO]: Saving and pausing game...
[21:42:42] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:42:42] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:42:42] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:42:42] [Server thread/INFO]: SaturnoSomaro92 lost connection: Disconnected
[21:42:42] [Server thread/INFO]: SaturnoSomaro92 left the game
[21:42:42] [Server thread/INFO]: Stopping singleplayer server as player logged out
[21:42:42] [Server thread/INFO]: Stopping server
[21:42:42] [Server thread/INFO]: Saving players
[21:42:42] [Server thread/INFO]: Saving worlds
[21:42:43] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:overworld
[21:42:45] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_nether
[21:42:45] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Juliet Willow]'/minecraft:the_end
[21:42:45] [Server thread/INFO]: ThreadedAnvilChunkStorage (Juliet Willow): All chunks are saved
[21:42:45] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[21:42:45] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[21:42:45] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[21:42:47] [Render thread/INFO]: Stopping!
```

### Other Mods

```shell
No other mods installed. Tried this alone as soon as i saw this.
```

**Comment by quiqueck** [comment]
Ah, thanks for the reminder. It was actually a Bug that the large willow trees would grow from saplings. :)

**Comment by quiqueck** [comment]
Released with BetterNether 7.1.2. Please re-open this issue if you still see odd behaviour.
